tests/Unit/Managers/TwitchManagerTest.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Tests\Unit\Managers;
+
+use App\Infrastructure\Clients\ApiClient;
+use App\Managers\TwitchManager;
+use App\Providers\TwitchTokenProvider;
+use Carbon\Carbon;
+use Illuminate\Http\Client\Response;
+use Mockery;
+use Tests\TestCase;
+use GuzzleHttp\Psr7\Response as GuzzleResponse;
+
+/**
+ * @SuppressWarnings(PHPMD.StaticAccess)
+ */
+class TwitchManagerTest extends TestCase
+{
+    protected $tokenProvider;
+    protected $apiClient;
+    protected $twitchManager;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->tokenProvider = Mockery::mock(TwitchTokenProvider::class);
+        $this->apiClient = Mockery::mock(ApiClient::class);
+        $this->twitchManager = new TwitchManager($this->tokenProvider, $this->apiClient);
+    }
+
+    protected function tearDown(): void
+    {
+        Mockery::close();
+    }
+
+    /**
+     * @test
+     */
+    public function fetchStreamsFromTwitch()
```

Comment:
Si on me confundo es más correcto
```suggestion
    public function fetchesStreamsFromTwitch()
```